src/qae/initial-properties.js

```diff
@@ -117,34 +117,98 @@ const properties = {
  */
 
 /**
- * General styling for all columns.
- * Split up into header and content (body) styling.
- * If any property is not set, default values specific for each property is used.
+ * Styling defintions
  * @name Component
+ * @type {ChartStyling | GeneralStyling}
+ */
+
+/**
+ * Mandatory key for general styling
+ * @name GeneralStylingKey
+ * @type {"general"}
+ */
+
+/**
+ * Font styling values
+ * @name FontStyleValues
+ * @type {"bold" | "italic" | "underline"}
+ */
+
+/**
+ * General chart styling
+ * @name GeneralStyling
+ * @type object
+ * @property {GeneralStylingKey} key
+ * @property {TitleStyling} title
+ */
+
+/**
+ * Title styling options
+ * @name TitleOptions
+ * @type object
+ * @property {string} [fontSize] - Font size in pixel value
```

Comment:
I don't recognize this syntax, are the [] to highlight the prop name?

Reply:
`@property {string} [fontSize]` means that `fontSize` is optional. Which is the JSDoc way of doing it. You can also do @property {string=} fontSize which would be the Google Closure compiler syntax, that also works.

Reply:
ah ok, didn't know that. I think I prefer this way